### PR TITLE
Add django-inmemorystorage to requirements

### PR DESCRIPTION
**Pull request by adam-thomas**
Didn't realise this was an external library!  @incuna/backend quick merge please? :)

### CODE REVIEW

**Comment by LilyFirefly** [comment]
LGTM, but I don't have write access to this repo to merge.

File: requirements.txt [hunk 1]
```diff
@@ -2,6 +2,7 @@
 -e .
 colour-runner==0.0.4
 coverage==3.7.1
+django-inmemorystorage==0.1.1
```
Comment (meshy) [inline requirements.txt:5]: use [`dj-inmemorystorage`](https://pypi.python.org/pypi/dj-inmemorystorage/1.4.0) instead

Comment (meshy) [inline requirements.txt]: https://pypi.python.org/pypi/dj-inmemorystorage/1.4.0

Comment (meshy) [inline requirements.txt:5]: (See https://github.com/waveaccounting/dj-inmemorystorage#differences)

Comment (adam-thomas) [inline requirements.txt:5]: :+1:

**Comment by adam-thomas** [comment]
@Ian-Foote You don't have write access?  WTF.

**Comment by adam-thomas** [comment]
Try now (well, after I've addressed @meshy's comment), I added @incuna/backend in the options.

**Comment by LilyFirefly** [comment]
Oops, I misread that.

**Comment by adam-thomas** [comment]
No worries, I need to add some extra coverage config anyway so I'll just do it in that PR :)

**Pull request merged**